client/services/FeedbackGeneratorService.js

```diff
@@ -280,6 +307,16 @@ tie.factory('FeedbackGeneratorService', [
     };
 
     /**
+     *
+     */
+    var _getUnfamiliarLanguageFeedback = function() {
+      return [
+        "Seems like you're having some trouble with this language. Why ",
+        "don't you take a look at the primer?"
```

Comment:
Perhaps we can add a small note to look at the bottom of the screen or to look at the `New to Python` button (or whatever we decide to call the primer button) so the user knows where to go.